packages/elements/src/Header/Header.tsx

```diff
@@ -191,11 +191,7 @@ export default function Header(props: Props) {
     : null;
 
   const headerTitle =
-    typeof customTitle !== 'function'
-      ? (props: React.ComponentProps<typeof HeaderTitle>) => (
-          <HeaderTitle {...props} />
-        )
-      : customTitle;
+    typeof customTitle === 'function' ? customTitle : HeaderTitle;
```

Comment:
this change should be reverted, since `HeaderTtitle` is a component and can't be called like a function